.github/workflows/storefront.yml

```diff
@@ -44,6 +44,40 @@ jobs:
           npm run lint:js
           npm run lint:scss
 
+  twig-lint-storefront:
+    name: Twig Lint (Storefront)
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Check for relevant changes
+        id: changed-files
+        uses: tj-actions/changed-files@v45
+        with:
+          files: |
+            src/Storefront/Resources/views/ludtwig-config.toml
+            src/Storefront/Resources/views/.ludtwig-ignore
+            src/Storefront/Resources/views/**/*.twig
+
+      - name: Run ludtwig linter
+        if: steps.changed-files.outputs.any_changed == 'true'
+        uses: addnab/docker-run-action@v3
+        with:
+          image: maltejanz/ludtwig:0.10.0
+          options: -v ${{ github.workspace }}:/ludtwig
+          run: |
+            cd ./src/Storefront/Resources/views
+            ludtwig .
```

Comment:
I'm pretty sure the shell runs with `-e` so if the return code is nonzero, it will exit here

Reply:
Just saw it's a custom image, well then it depends on the image :)

Reply:
And btw: I introduduced a new workflow `addnab/docker-run-action@v3` here, because that was the easiest way to get the old behaviour from GitLab translated to GitHub. I tried using a container for the whole job before but that fails the `tj-actions/changed-files@v45` workflow because the container is based on alpine and doesn't contain for example git...

Reply:
Thanks for the context 👍 I guess it's just an convenient way of doing a docker run